migrations/20210914160014_group_extensions.js

```diff
@@ -0,0 +1,35 @@
+exports.up = async function(knex) {
+    await knex.schema.createTable('extensions', table => {
```

Comment:
hmm, why does the indentation seem off for all these files? something with your setup? should all be two spaces not 4